templates/common/_base/units/crio.service-kubens.yaml

```diff
@@ -0,0 +1,18 @@
+# Autogenerated by hack/update-templates.sh; do not edit
```

Comment:
does this being here mean we'll have the kubens enabled by default? is that intended?

Reply:
This drop-in does nothing except set the environment variable that will be laid down by kubens.service if it's running, so it's safe fro this to be present all the time.

The kubens.service itself is not being enabled by default (see https://github.com/openshift/machine-config-operator/pull/3274/files/d8f42ae7a8e66970a5fcbdaac6fd8712d1466ba7#diff-c38a0be931e5bc991f959879c6af89aee96be94967b4544f63ace1d52e5f88ccR3)